### DISCUSSION TITLE
Possible issue with $.native giving Semantics handle error

### DISCUSSION THREAD

**Discussion by benjamin-inspectly** [leancodepl/patrol#1343] [Q&A] [answered]
Hi all,

I didn't want to waste the devteams time if it's just me having some trouble - so I'm asking here instead of creating an issue just yet. Hope that's okay.

I believe I set up everything for native automation with Android. Everything is working just fine using the commands and I'm able to use both "pressHome" and handle permissions without any problems.

The problem I have though, is that even though my test completes every step as expected - it keeps giving me a failed test result.
When opening the test results report I get this error:

![image](https://github.com/leancodepl/patrol/assets/114057836/655ea6f5-6913-423f-9210-c611ef575166)

I found out that if I do not use the native features it returns "Pass" every time, but the second I use $.native as below, my test fails.

![image](https://github.com/leancodepl/patrol/assets/114057836/d3905ed7-71c3-426a-9247-3ee79769d9ea)

Do any of you know if I forgot to do something in the setup process? I believe I did everything correctly, but I might be wrong?

Patrol doctor:
![image](https://github.com/leancodepl/patrol/assets/114057836/db466c43-a6ef-41da-a2ba-d2e4f46cdbf4)

Flutter doctor:
![image](https://github.com/leancodepl/patrol/assets/114057836/bc0bb8f4-f676-4509-b60c-723963dfc9ba)

**Comment by bartekpacia** [comment, marked as answer]
Oopsie, I must've missed this Discussion - sorry! Anyway, this is [a known bug](https://github.com/leancodepl/patrol/issues/1265) and has been fixed in [`patrol v1.1.5`](https://pub.dev/packages/patrol/changelog#115)

**Reply by benjamin-inspectly** [reply]
Thank you so much! I will test it immediatly.
